docker-compose-dev.yml

```diff
@@ -22,28 +26,33 @@ services:
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-flowflow}
       - FM_PASSWORD=${FM_PASSWORD:-foo}
       - API_PASSWORD=${API_PASSWORD:-foo}
+      - CACHE_SIZE=${CACHE_SIZE:-""}  # if empty, defaults to using 10% of the available disk space
+      - CACHE_HALF_LIFE=${CACHE_HALF_LIFE:-1000}
     shm_size: 1G
     tty: true
     stdin_open: true
     restart: always
-    volumes:
-      - data_volume_flowdb:/var/lib/postgresql/data
```

Comment:
I'd put this back I think, purely because the upstream Postgres image creates a volume that maps to this dir _anyway_, and this at least makes it explicit.

Reply:
The reason I took the flowdb volume out is because if it is present then for flowdb to pick up source code changes it is _not_ sufficient to simply re-build the docker image and restart the container. In addition, you also need to remember to delete the volume before restarting the container, which is easy to forget and has led to lots of confusion and long debugging sessions in the past. Therefore I deemed it safer to remove the volume.